db/artillery.yml

```diff
@@ -0,0 +1,19 @@
+config:
+  target: "http://127.0.0.1:3006"
+  phases:
+    - duration: 120
+      arrivalRate: 10
+
+scenarios:
+  - name: "Look at a random project"
+    flow:
+      - loop:
+        - get:
+            url: "/api/community/{{ $randomNumber(1,10000000) }}"
+        count: 100
+  - name: "Look at one of top 10000 projects"
+    flow:
+      - loop:
+        - get:
+            url: "/api/community/{{ $randomNumber(1,10000) }}"
+        count: 100
```

Comment:
one thing you could have done is use scenario weights, which allow you to specify relatively how often different scenarios should be run (weight of 3/4 run 75% of the time)

Reply:
My goal for this was 50-50 between the two scenarios, so I left off scenario weights here; if I fine-tune, I'll incorporate them.